onnx_coreml/_backend_rep.py

```diff
@@ -7,11 +7,13 @@
 from typing import Any, Sequence, List
 from onnx.backend.base import BackendRep, namedtupledict
 from coremltools.models import MLModel  #type: ignore
+from typing import Dict, Any, Text
 
 
 class CoreMLRep(BackendRep):
     def __init__(self,
                  coreml_model,  # type: MLModel
+                 onnx_outputs,  # type: Dict[Text, Any]
```

Comment:
Do we know a more concrete type than `Any` for an output? If it's a shape, something like `Tuple[int, ...]` might work.

Reply:
Yes Tuple[int, ...] does work